kubernetes/structures_pod.go

```diff
@@ -313,6 +313,32 @@ func flattenSecretVolumeSource(in *v1.SecretVolumeSource) []interface{} {
 
 // Expanders
 
+func expandPodSecurityContext(l []interface{}) *v1.PodSecurityContext {
```

Comment:
Why was it necessary to move this functions's code ?

If this is just cosmetic, I'd rather have it where it was for the sake of preserving git history.
Moving it around will make it harder to drill down into the history of its changes.